docs/30DaysOfPWA/advanced-capabilities/06.md

```diff
@@ -1,15 +1,174 @@
+What's about the web? Can we keep installed web frontend application and data it uses up to date on our users' devices? Yes! With the API called Periodic Background Sync API, we can ask the background service of the browser to run parts of our code with some periodicity regardless of the main application status (whether it's running or not). Like Background Sync, this API only provides an event to the service worker - you have to implement the actual "payload" yourself. For example, it could be a fetch to check for the updated app version and/or its content followed by corresponding resources update in the browser cache.
+
+Let's see how it works for a web-based social media client that can periodically update its main feed content in the background.
+
+### How to start
+
+It's a really good idea to let the user decide if they want to use this feature or not. Using `PeriodicSyncManager` interface of the registered service worker you can list existing "periodic syncs" to set the value (on/off) of UI controls (switches, checkboxes, etc) labeled like "Update feed in the background". When the user turns on this feature, you should register a new periodic sync:
+
+```javascript
+async function registerPeriodicFeedUpdate() {
+  const registration = await navigator.serviceWorker.ready;
+
+  // Query and check permission. See "Privacy and resource utilization considerations" section below.
+  const status = await navigator.permissions.query({
+    name: 'periodic-background-sync',
+  });
+  if (status.state !== 'granted') {
+  {
+    console.log('Periodic Background Sync is not granted.');
+    return;
+  }
+
+  try {
+    await registration.periodicSync.register('update-feed-content', {
+      minInterval: 24 * 60 * 60 * 1000, // We ask browser to run the sync no more than once a day
+    });
+    showNotification('Success! Feed will be updated in the background.');
+
+  } catch() {
+    console.error('Periodic Background Sync registration failed');
+  }
+}
+```
+
+Like always in PWA, feature detection is a best practice:
+
+```javascript
+if ('serviceWorker' in navigator && 'PeriodicSyncManager' in window) {
+   registerPeriodicFeedUpdate(post)
+} else {
+  console.log('Periodic Background Sync is not supported');
+}
+```
+
+In the service worker, you should listen and react to the `periodicsync` event named `update-feed-content`:
+
+```javascript
+self.addEventListener('periodicsync', event => {
+  if (event.tag === 'update-feed-content') {
+      event.waitUntil(
+
+        // You have to implement the function that fetches the latest posts and updates the storage:
+        updateFeedContent()
+      );
+  }
+});
+```
+
+Please note, that it's not an obligation but a possibility for you to fetch new data in `updateFeedContent` function. Before sending a server request, you might want to check if the user is on a non-metered connection and if there is enough space in the browser storage using Network Information and StorageManager APIs respectively.
```

Comment:
If we have other days in the blog series that talk about these APIs, then let's link to them. If not, maybe link to MDN or any other relevant place where people can learn more.

Reply:
`Good idea. Will add links to MDN`